tests/test_config.py

```diff
@@ -1055,9 +1055,12 @@ def test_set_forced_variation__invalid_variation_key(self):
 
     self.assertFalse(self.project_config.set_forced_variation('test_experiment', 'test_user',
                                                               'variation_not_in_datafile'))
-    self.assertTrue(self.project_config.set_forced_variation('test_experiment', 'test_user', ''))
     self.assertTrue(self.project_config.set_forced_variation('test_experiment', 'test_user', None))
 
+    with mock.patch.object(self.project_config, 'logger') as mock_config_logging:
+      self.assertIs(self.project_config.set_forced_variation('test_experiment', 'test_user', ''), False)
```

Comment:
nit. Use `self.assertFalse` instead.

Reply:
assertFalse asserts Falsy value which includes None. assert_is ensures that we are returning strictly False from the method.